DoSomething/gladiator
# Pull from phoenix

#### What's this PR do?

Pull campaign info from phoenix into gladiator.

Many TODOs here:

* store campaigns in cache and grab info from there.
* Bulk grab a set of campaigns from phoenix for pages like the contest.index view 

#### How should this be manually tested?
pull down, any where you used to see an campaign ID, you should see a real campaign title

#### Any background context you want to provide?

Need to get this in so we can start pulling in campaign content into emails. There is still much to do in terms of getting this code right. 

#### What are the relevant tickets?
Fixes #14
